docs/meeting_notes/2024.md

```diff
@@ -1,5 +1,46 @@
 # Meeting notes from 2024
 
+## 27 June 2024
+
+Ad-hoc meeting to make progress on governance PR 226
+
+### Attendees
+* Ana Ruvalcaba
+* Jason Grout
+
+### Notes
```

Comment:
@Ruv7 @Zsailer - we need to fill out the minutes from today here.

Reply:
I think I did this but I also can't figure out how to see a preview. Let's review in our next meeting. Thanks.